File: allOrNothing.js
function possiblyPerfect(key,answers) {
    let correct = 0;
    let unknown = 0;
    let wrong = 0;
    for (let i = 0; i < answers.length; i++){
         if (key[i] === answers[i]){
      correct += 1;
    }else if (key[i] === '_'){
      unknown += 1;
    }else{
      wrong += 1;
    }
    }
  if (correct + unknown === answers.length || wrong + unknown == answers.length){
    return true;
  }else{
    return false;
  }
  }

File: all_or_nothing.py
def possibly_perfect(key, answers):
    correct = 0
    wrong = 0
    unknown = 0
    for x in range(len(answers)):
        if key[x] == answers[x]:
            correct += 1
        elif key[x] == '_':
            unknown += 1
        else:
            wrong += 1
    if (correct + unknown) == len(answers) or (wrong + unknown) == len(answers):
        return True
    else:
        return False
    